### ISSUE TITLE
PosteriorProbabilitiesUtils bug?

### ISSUE THREAD

**Issue by ldgauthier** [broadinstitute/gatk#4760]
One of the example commands in CalculateGenotypePosteriors describes the usage of the `-supporting` argument and the `--num-reference-samples-if-no-call` argument at the same time:
```
 gatk --java-options "-Xmx4g" CalculateGenotypePosteriors \
   -V input.vcf.gz \
   -O output.vcf.gz \
   -supporting 1000G.phase3.integrated.sites_only.no_MATCHED_REV.hg38.vcf.gz \
   --num-reference-samples-if-no-call 2504
```
Calculate the posterior genotypes of a callset, and impose that a variant *not seen* in the external panel is tantamount to being AC=0, AN=5008 within that panel

We don't have any tests that use both of these arguments at the same time, but it looks like the behavior in that case is wrong.  PosteriorProbabilitiesUtils adds numRefSamplesFromMissingResources regardless of whether there was an overlapping variant in the panel or not, effectively diluting the AF of all the variants used as priors and making the number of reference alleles very inconsistent across variants.

**Comment by ldgauthier** [comment]
Maybe @davidbenjamin will serve as another set of eyes?  I have a small fleeting hope that I'm reading something wrong.

**Comment by davidbenjamin** [comment]
>PosteriorProbabilitiesUtils adds numRefSamplesFromMissingResources regardless of whether there was an overlapping variant in the panel or not, effectively diluting the AF of all the variants used as priors and making the number of reference alleles very inconsistent across variants.

@ldgauthier That's exactly what it looks like to me, too.

**Comment by ldgauthier** [comment]
I touched it last so I will fix it.

**Comment by davidbenjamin** [comment]
@ldgauthier You fixed this in #4793, didn't you?

**Comment by ldgauthier** [comment]
Yes!  closed via #4793
